code/modules/client/preference_setup/loadout/loadout_factions.dm

```diff
@@ -110,10 +110,10 @@
 
 /datum/gear/faction/zenghu_labcoat/New()
 	..()
-	var/masks = list()
+	var/list/masks = list()
```

Comment:
The var name doesn't quite fit, but it is fairly minor I suppose

Reply:
i blame contributors